### PR TITLE
Fix for seeding script name errors

### PR DESCRIPTION
**Pull request by Braagaa**
**What it does**

- Simple regex solution to the numbers appearing in name sometimes during the seeding script

### CODE REVIEW

**Comment by dproc96** [comment]
@Braagaa Did you add the latest changes you mentioned in Slack? I don't see a fix in here that accommodates apostrophes or hyphens in the names

File: server/scripts/dbSeeding.js [hunk 1]
```diff
@@ -55,8 +55,8 @@ const mainCitiesGPS = [Toronto, QuebecCity, Montreal, Calgary];
       }
 
       const mock = {
-        firstName: name.firstName(),
-        lastName: name.lastName(),
+        firstName: name.firstName().replace(/[0-9]|'/g, ""),
+        lastName: name.lastName().replace(/[0-9]|'/g, ""),
```
Comment (Braagaa) [inline server/scripts/dbSeeding.js]: @dproc96 these two line

**Review by Braagaa — COMMENTED** [review]
I ran it 20 times with no error lol

File: server/scripts/dbSeeding.js (hunk 1 above)
Comment (Braagaa) [reply server/scripts/dbSeeding.js]: The regex will look for any number and ' and replace with an empty string

Comment (dproc96) [reply server/scripts/dbSeeding.js]: Why are we removing apostrophes? Shouldn't we allow users to have an apostrophe in their name?

Comment (Braagaa) [reply server/scripts/dbSeeding.js]: I see what you mean. Let me look into it.

Comment (Braagaa) [reply server/scripts/dbSeeding.js]: Hey what do you think of the solution?

**Review by dahrk — APPROVED** [review]
When would we see numbers in the names?

**Comment by Braagaa** [comment]
Faker.js sometimes puts numbers in there random names for some reason

**Pull request merged**